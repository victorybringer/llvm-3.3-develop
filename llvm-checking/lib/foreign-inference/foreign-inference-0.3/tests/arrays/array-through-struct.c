struct S {
  int ** fld;
  double d;
};

extern int x;

int f(int **arr1, double * arr2) {
  struct S s;
  s.fld = arr1;
  s.d = arr2[6];
  return s.fld[3][x] / s.d;
}
