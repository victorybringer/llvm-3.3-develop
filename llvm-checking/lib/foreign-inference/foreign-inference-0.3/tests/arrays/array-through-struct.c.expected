fromList [ (("@f", "%arr1"), 2)
         , (("@f", "%arr2"), 1)
         ]
